python/lsst/daf/butler/core/location.py

```diff
@@ -255,8 +277,15 @@ def _fixupFileUri(parsed, root=None, forceAbsolute=False):
         to be turned into absolute paths before they can be used.  This is
         always done regardless of the ``forceAbsolute`` parameter.
 
+        AWS S3 differentiates between keys with trailing POSIX separators (i.e
+        `/dir` and `/dir/`) whereas POSIX does not neccessarily. This must be
+        tracked manually if network location is to be treated interchangeably
```

Comment:
Because `_fixupFileUri` is a static method I wanted to make it clear that anyone that uses it as a `ButlerURI._fixupFileUri(parsed)` needs to be aware of that fact and consider whether the returned `dirLike` flag is important for them or not.

Reply:
Please remove the final sentence. I still find it confusing and we already know this is a static method so people need to track the return values.

Reply:
Removed.